Shoko.Server/Server/ShokoServer.cs

```diff
@@ -1023,23 +1016,15 @@ public static void ScanDropFolders()
     }
 
     public static void ScanFolder(int importFolderID)
-    {
-        if (!workerScanFolder.IsBusy)
-        {
-            workerScanFolder.RunWorkerAsync(importFolderID);
-        }
-    }
-
-    public static void RunImport()
-    {
-        Analytics.PostEvent("Importer", "Run");
-
-        if (!workerImport.IsBusy)
+    { 
+        var schedulerFactory = ServiceContainer.GetService<ISchedulerFactory>();
+        var scheduler = schedulerFactory!.GetScheduler().Result;
+        scheduler.TriggerJob(ScanFolderJob.Key, new JobDataMap
         {
-            workerImport.RunWorkerAsync();
-        }
+            {"importFolderID", importFolderID}
```

Comment:
TODO: cleanup and publish my job factory implementation as a nuget and use it here

Reply:
`scheduler.StartJob(JobBuilder<ScanFolderJob>.Create().UsingJobData(a => a.ImportFolderID = importFolderID).WithGeneratedIdentity().Build()).Result;`